### PR TITLE
Trial code review (Microverse's trials code"

### PR DESCRIPTION
**Pull request by samwel-mongare**
Added the index.html file together with its styling file style.css into the repo. Doing a review of the code written in collaboration session in the microverse application process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# Trials-code-review
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (Mosams) [inline README.md]: Code functionality looks good 🔥 
No linters errors 🏅

Comment (samwel-mongare) [inline README.md]: I suggest the implimentation of a more comprehensive readme.md file with more details of the project and author

File: index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            <label for="email" id="email-label">Email: &nbsp;</label><br>
+            <input type="email" id="email" placeholder="Enter your email" required="email"><br>
+
+            <label for="number" id="number-label">Age: &nbsp;</label><br>
+            <input type="number" id="number" placeholder="Enter age btwn 18 and 40" min="18" max="40" class="form-inputs" required> </input>
+            <br>
+
+            <label for="dropdown" id="number">Select Months of partcipation: &nbsp;</label>
+            <label for="dropdown" id="number">
+
```
Comment (samwel-mongare) [inline index.html:29]: I suggest proper indentation of the code, avoid unnecessary spacing in you code. This will help improve on the readability and presentation of the code. Spacing is usually convenient for separating large sections of your code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+			</select>
+			<br>
+			<br>
+		
+
+			<label for="question"> DO YOU RECOMMEND MICROVERSE TO YOUR FRIENDS? </label><br>
+
+
+            <input id="question" type="radio" id="black" name="color" value="black">
+            <label for="question"> YES,Of Course</label> <br>
```
Comment (samwel-mongare) [inline index.html:43]: Another example of unnecessary spacing in the code

**Review by samwel-mongare — COMMENTED** [review]
Keep doing the  good work 🏅 
No linters errors 👍 
Code functions as intented 🔥 

Keep rocking 🚀

File: index.html (hunk 2 above)
Comment (omar-labana) [reply index.html:29]: - [ ] Good job highlighting the issue politely, please rephrase your reasoning elaborating why the reviewer should avoid these pitfalls/flaws in his/her code, does it reduce the readability of the code? or does affect a certain metric?

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Mosams, 

Excellent work so far! 👌
You have used GitHub features for reviews and how to communicate in a professional way. ✔️
You phrased the comments politely as suggestions.  ✔️
There are some minor issues that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the code you have reviewed._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (omar-labana) [reply README.md]: - [ ] Great work on that one, you can rephrase it to be a bit passive suggestion.

File: index.html (hunk 2 above)
Comment (omar-labana) [reply index.html:29]: - [ ] Nice work on highlighting the issue politely, please rephrase your reasoning, elaborate more on why are you requesting this change, does it reduce the readability of the code? does it affect a certain metric required?

**Review by samwel-mongare — COMMENTED** [review]
Changes and more explanation on the review. I have added my reasoning to the review as requested

**Comment by omar-labana** [comment]
Hi @Mosams, 

Your project is complete! Excellent work improving your communication providing feedback! :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
